src/qcodes/instrument_drivers/Keysight/Keysight_N9030B.py

```diff
@@ -39,7 +39,9 @@ def get_raw(self) -> ParamRawDataType:
 
 
 class Trace(ParameterWithSetpoints):
-    def __init__(self, number: int, *args: Any, **kwargs: Any) -> None:
+    def __init__(
+        self, number: int, *args: Any, get_data: Callable, **kwargs: Any
```

Comment:
```suggestion
        self, number: int, *args: Any, get_data: Callable[[int], ParamRawDataType], **kwargs: Any
```
I think this can be made more specific